Castle.Transactions.sln

```diff
@@ -5,6 +5,7 @@ VisualStudioVersion = 15.0.27703.2000
 MinimumVisualStudioVersion = 10.0.40219.1
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{B358A960-356B-47CD-8E66-C275F15BF893}"
 	ProjectSection(SolutionItems) = preProject
+		buildscripts\common.props = buildscripts\common.props

```

Comment:
This should not be here. I was attempting to re-platform your build so that is has parity with Core/Windsor. Will remove!